packages/vx-responsive/src/components/ParentSize.tsx

```diff
@@ -62,6 +62,7 @@ export default class ParentSize extends React.Component<
   componentWillUnmount() {
     if (this.animationFrameID) window.cancelAnimationFrame(this.animationFrameID);
     if (this.resizeObserver) this.resizeObserver.disconnect();
+    if (this.resize) this.resize.cancel();
```

Comment:
Is the `if (...)` statement needed here? since the debounced method is defined in the constructor I think this check is unneeded (versus `animationFrameID`/`resizeObserver` which can be `null` or `undefined`)